### PR TITLE
style: simplify test assertions

### PR DESCRIPTION
**Pull request by bvandrc**

### CODE REVIEW

File: packages/table/test/quadrants/tableQuadrantTests.tsx [hunk 1]
```diff
@@ -144,19 +144,19 @@ describe("TableQuadrant", () => {
...
                 const component = mountTableQuadrant({ menuRenderer });
                 expect(menuRenderer.called).to.be.true;
-                expect(component.find(`.${Classes.TABLE_TOP_CONTAINER} > .${MENU_CLASS}`).length).to.equal(1);
+                expect(component.find(`.${Classes.TABLE_TOP_CONTAINER} > .${MENU_CLASS}`)).to.have.lengthOf(1);
             });
 
             it("does not render menu if menuRenderer not provided", () => {
                 const component = mountTableQuadrant();
-                expect(component.find(`.${Classes.TABLE_TOP_CONTAINER}`).children().length).to.equal(0);
+                expect(component.find(`.${Classes.TABLE_TOP_CONTAINER}`).children()).to.have.lengthOf(0);
```
Comment (ggdouglas) [inline packages/table/test/quadrants/tableQuadrantTests.tsx]: **nit:** `.to.be.empty` seems a bit more appropriate for this type of check:
```suggestion
                expect(component.find(`.${Classes.TABLE_TOP_CONTAINER}`).children()).to.be.empty;
```

File: packages/table/test/regionsTests.ts [hunk 2]
```diff
@@ -238,7 +238,7 @@ describe("Regions", () => {
         ];
 
         const joinedGroups = Regions.joinStyledRegionGroups([Regions.row(2)], myGroups, undefined);
-        expect(joinedGroups).to.have.lengthOf(2);
+        expect(joinedGroups).to.have.to.have.lengthOf(2);
```
Comment (ggdouglas) [inline packages/table/test/regionsTests.ts]: **nit:** duplicate chain for assertions in this file:
```suggestion
        expect(joinedGroups).to.have.lengthOf(2);
```

**Review by ggdouglas — COMMENTED** [review]
A couple small comments, but the reset generally LGTM

File: packages/table/test/quadrants/tableQuadrantTests.tsx (hunk 1 above)
Comment (bvandrc) [reply packages/table/test/quadrants/tableQuadrantTests.tsx]: Sure, done!

File: packages/table/test/regionsTests.ts (hunk 2 above)
Comment (bvandrc) [reply packages/table/test/regionsTests.ts]: Good catch. Fixed

**Review by ggdouglas — APPROVED** [review]
Nice work, thanks for the cleanup!

**Pull request merged**